dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python
```

Comment:
Fyi I updated my local `Dockerfile`

```python
FROM python:3.6
```

which I believe the version that maap api nasa prod is using

Reply:
```suggestion
FROM python:3.6
```